relate/settings.py

```diff
@@ -306,4 +306,14 @@
 
 # }}}
 
+# {{{ Enable auto reload when local_settings changes when debugging
+
+if globals().get("DEBUG", False):
+    try:
+        import local_settings  # type: ignore
```

Comment:
Why not replace the `open`/`exec` logic above with an import?

Reply:
I don't know that's also OK.

Reply:
> I don't know that's also OK.

Why wouldn't it be? It requires a little rewiring, but it's probably better anyhow.

Reply:
The one issue perhaps is to make sure that it can be found by the `import`.

Reply:
> The one issue perhaps is to make sure that it can be found by the `import`.

Yes. I don't know how to make it working in that way. We need to import the local settings by name, since it doesn't have a consistent name for runserver or runtests.  `importlib.import_module` can import a module by name, but with the prerequisit that the dirctory has an `__init__.py`, which is missing in our case.